src/node.cc

```diff
@@ -43,6 +43,10 @@
 #include "v8-profiler.h"
 #include "zlib.h"
 
+#ifdef NODE_ENABLE_VTUNE_PROFILING
+#include "../deps/v8/src/third_party/vtune/v8-vtune.h"
```

Comment:
Looks like the linter was [complaining](https://ci.nodejs.org/job/node-test-linter/305/console) about this line.  Can you run `make lint` and add a `// NOLINT` comment if it's unhappy?